src/tm_mad/ceph/postmigrate

```diff
@@ -55,6 +70,8 @@ fi
 
 migrate_other "$@"
 
-exec_and_log "$SSH $SRC_HOST rm -rf $DST_PATH"
+if [ "$SHARED" != "YES" ]; then
```

Comment:
I think it is OK because there are other places with similar code style. Take for example

https://github.com/OpenNebula/one/blob/master/src/tm_mad/ssh/mv#L62